libs/hiraya/src/components/Button/style.ts

```diff
@@ -0,0 +1,42 @@
+import { Button as MUIButton, styled } from '@mui/material';
+
+import { variantStyle, buttonSizeStyle } from './theme';
+
+interface ButtonProps {
+  label?: string;
+  variants?: keyof typeof variantStyle;
+  buttonHeightSizes?: keyof typeof buttonSizeStyle;
+  isFullWidth?: boolean;
+  iconOnly?: boolean;
+}
+
+const ButtonStyled = styled(MUIButton)<ButtonProps>(
+  ({ variants, buttonHeightSizes, isFullWidth, iconOnly }) => ({
+    backgroundColor: variantStyle[variants || 'primary'].background,
+    color: variantStyle[variants || 'primary'].text,
+    padding: '0',
+
+    '&.MuiButton-root': {
+      minWidth: iconOnly
+        ? buttonSizeStyle[buttonHeightSizes || 'large']
+        : !isFullWidth
+        ? '92px'
+        : '112px',
```

Comment:
```suggestion
        : isFullWidth
        ? '112px'
        : '92px',
```

Let's avoid negating the expression to make it more readable on ternary conditions.